addons/website_slides/views/website_slides_profile.xml

```diff
@@ -0,0 +1,148 @@
+<?xml version="1.0" ?>
+<odoo>
+    <data>
+        <template id="profile_header" name="Slides Profile Header">
```

Comment:
Guidelining note (check other templates): define <odoo><data> on a one liner to save indentation :) . It is easier to read / maintain.